app/src/main/java/de/danoeh/antennapod/activity/OpmlImportBaseActivity.java

```diff
@@ -19,22 +30,23 @@
     private static final String TAG = "OpmlImportBaseActivity";
     private OpmlImportWorker importWorker;
 
-    /**
+	private static final int PERMISSION_REQUEST_READ_EXTERNAL_STORAGE = 5;
```

Comment:
It is like the requestCode in onActivityResult [1]
Not sure what the best practice is, but magic constants (using ``0``) is probably not the best idea.

--
[1] http://developer.android.com/intl/ko/reference/android/app/Activity.html#onActivityResult(int, int, android.content.Intent)